tests/foreman/cli/test_hostgroup.py

```diff
@@ -669,6 +681,89 @@ def test_negative_create_with_content_source(self):
                 'organization-ids': self.org['id'],
             })
 
+    @run_only_on('sat')
+    @tier2
+    @upgrade
+    def test_positive_create_with_synced_content(self):
+        """Check if hostgroup with synced kickstart repository can be created
+
+        :id: 7c51ac72-359c-488a-8658-88b5a94d7e7a
+
+        :expectedresults: Hostgroup should be created and has proper
+            installation content id present
+
+        :BZ: 1415707
+
+        :CaseLevel: Integration
+        """
+        # Common entities
+        org = make_org()
+        lce = make_lifecycle_environment({'organization-id': org['id']})
+        product = make_product({'organization-id': org['id']})
+        repo = make_repository({
+            u'url': settings.rhel6_os,
+            u'product-id': product['id'],
+            u'content-type': u'yum',
+        })
+        Repository.synchronize({'id': repo['id']})
+
+        cv = make_content_view({
+            'organization-id': org['id'],
+            'repository-ids': [repo['id']],
+        })
+        ContentView.publish({'id': cv['id']})
+        cv = ContentView.info({'id': cv['id']})
+        cvv = cv['versions'][0]
+        ContentView.version_promote({
+            'id': cvv['id'],
+            'to-lifecycle-environment-id': lce['id'],
+        })
+
+        # Get the Partition table ID
+        ptable = PartitionTable.info({'name': DEFAULT_PTABLE})
+
+        # Get the arch ID
+        arch = Architecture.list({
+            'search': 'name={0}'.format(DEFAULT_ARCHITECTURE)})[0]
+        # Get the OS ID
+        os = OperatingSys.list({
+            'search': 'name="RedHat" AND major="{0}" OR major="{1}"'.format(
+                RHEL_6_MAJOR_VERSION, RHEL_7_MAJOR_VERSION)
+        })[0]
+
+        # Update the OS with found arch, ptable, templates and media
+        OperatingSys.update({
+            'id': os['id'],
+            'architectures': arch['name'],
+            'partition-tables': ptable['name'],
+        })
+        proxy = Proxy.list({
+            'search': 'url = https://{0}:9090'.format(settings.server.hostname)
+        })[0]
+
+        repos_list = Repository.list({
+            'content-view-version-id': cvv['id'],
+            'organization-id': org['id'],
+        })
+        synced_repo = max(repos_list, key=lambda x: x['id'])
```

Comment:
we have created one org, one content view , one repo , why we should expect many repos here?